### ISSUE TITLE
Inconsistent Coloring in Involvements/Faculty Pages

### ISSUE THREAD

**Issue by Dopey12** [gordon-cs/gordon-360-ui#2305]
There are some messy color choices that lead to some insufficient color contrasts on multiple pages.
- Involvements Club Page

![Image](https://github.com/gordon-cs/gordon-360-ui/assets/123050501/8ae7ea8b-abda-425c-8691-150e6cc63b87)

![Image](https://github.com/gordon-cs/gordon-360-ui/assets/123050501/6694844a-087d-44d0-a779-c0829dbbdf03)

- Faculty/Staff Profiles

![Image](https://github.com/gordon-cs/gordon-360-ui/assets/123050501/63b70c3e-88cd-4a90-a4dc-11677973bda3)
